apps/public-docsite-v9/src/Concepts/WebComponentsInterop/UsingFluentReactWithWebComponents.stories.mdx

```diff
@@ -0,0 +1,148 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Developer/Web Components Interop/Using Fluent React with Web Components" />
+
+# Using Fluent React with Web Componenets
+
+Fluent React v9's extensible architecture and use of web platform features like CSS custom properties
+allow it to be extended to work well with [Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components),
+particularly [shadow DOM](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_shadow_DOM).
+
+Where it is necessary to modify Fluent React's default behavior to better interoperate with Web Components, the
+implementation will live in the [Fluent UI Contrib repository](https://github.com/microsoft/fluentui-contrib).
+
+## Rendering Fluent React in shadow DOM
+
+Fluent React components can be rendered inside shadow DOM with styles being set on the shadow root's `adoptedStyleSheets`
+property.
+
+```tsx
+import { root } from '@fluentui-contrib/react-shadow';
+import { FluentProvider, webLightTheme, Button } from '@fluentui/react-components';
+
+<FluentProvider theme={webLightTheme}>
+  {/* This is the shadow root */}
+  <root.div>
+    <Button>Fluent React Button in shadow DOM</Button>
+  </root.div>
+</FluentProvider>;
+```
+
+In the above example, note that `FluentProvider` sits outside the shadow DOM. When `FluentProvider` is inside the shadow
+DOM styling/theming will not work as expected.
+
+```tsx
+// ❌ This will not render correctly for example purposes only ❌
+import { root } from '@fluentui-contrib/react-shadow';
+import { FluentProvider, webLightTheme, Button } from '@fluentui/react-components';
+
+/* This is the shadow root */
+<root.div>
+  <FluentProvider theme={webLightTheme}>
+    <Button>Fluent React Button in shadow DOM</Button>
+  </FluentProvider>
+</root.div>;
+```
+
+To render a provider inside shadow DOM, use `ThemelessFluentProvider` instead.
+
+## `ThemelessFluentProvider`
```

Comment:
this PR should not be merged until this is released right ? https://github.com/microsoft/fluentui-contrib/pull/149

Reply:
Yep, as noted in the description 🙂

I moved it from draft for visibility since there was a fair bit of feedback on my recent documentation PRs.

Reply:
`ThemelessFluentProvider` is merged now. @layershifter when can we release it? I don't think I have access to the contrib release pipeline.

Reply:
Thanks @layershifter !